docs/provider/static.rst

```diff
@@ -30,30 +31,86 @@ already running resources.
                   # during the deployment (e.g HAProxy)
                   # At least 5 IPs routable through the
                   # APIs interface
+                  - 198.51.100.49
+                  - 198.51.100.50
+                  - 198.51.100.51
+                  - 198.51.100.52
+                  - 198.51.100.53
        # Openstack external network will be configured according
-       # to the following keys/values. Ips have to be routable through
+       # to the following keys/values. IPs have to be routable through
        # the external interface
-       start:   # First available ip
-       end:     # Last available ip
-       cidr:    # cidr notation to describe the pool of ips
-       gateway: # Ip address of the gateway
-       dns:     # Ip address of the DNS
+       start:   203.0.113.17     # First available IP
+       end:     203.0.113.31     # Last available IP
+       cidr:    203.0.113.16/28  # cidr notation to describe the pool of IPs
+       gateway: 203.0.113.254    # IP address of the gateway
+       dns:     203.0.113.253    # IP address of the DNS
      eths:
        - eth1   # Name of the APIs interface
        - eth2   # Name of the external interface
 
    resources: # An object with roles and there associated resources
      control:   # The role control, followed by resource info:
-       address:   # Ip of the control node
-       alias:     # Name for the node (optional)
-       user:      # ssh user name (optional)
-       keyfile:   # path to ssh private key (optional)
-       port:      # ssh port (optional)
+       address:   192.0.2.17  # IP of the control node
+       alias:     control   # Name for the node (optional)
+       user:      user      # ssh user name (optional)
+       keyfile:   keyfile   # path to ssh private key (optional)
+       port:      22        # ssh port (optional)
      compute:   # A role can also have a list of resources
-       - address: # Ip address of the first compute
-         alias:   # ...
-         user:    # ...
-         keyfile: # ...
-         port:    # ...
-       - address: # Ip address of the second compute
-       - address: # Ip address of the third compute
+       - address: 192.0.2.33  # IP address of the first compute
+         alias:   compute1  # ...
+         user:    user      # ...
+         keyfile: keyfile   # ...
+         port:    22        # ...
+       - address: 192.0.2.34  # IP address of the second compute
+       - address: 192.0.2.35  # IP address of the third compute
+
+All the hosts must have an IP address already configured on the APIs
+interface matching the the networks defined in ``reservation.yml``
+under ``network``, for example: 198.51.100.17 for the ``control``
+host, 198.51.100.33 for the ``compute1`` host and so on.
+
+In the ``inventory`` file there must be an host group entry for each
```

Comment:
Enos generates an inventory for you. It takes the [inventory.sample](https://github.com/BeyondTheClouds/enos/blob/2c7e4f775648a2bcf5aa96e81464d3d304aeca86/inventories/inventory.sample) template (or whatever template you specified into `reservation.yaml::inventory`) and appends `[control]`, `[compute]`, `[network]`, ... groups based on information provided in the `reservation.yaml`. So you don't have to build the inventory manually. 

Let me make this clearer in the general enos documentation so that you can remove the rest of this section.

Reply:
Hi @dancn, I just want let you know that I updated the documentation in #191.

Reply:
@dancn, @rcherrueau this is now online.

Reply:
```Enos generates an inventory for you. It takes the inventory.sample template (or whatever
template you specified into reservation.yaml::inventory) and appends [control], ```

With my request #194 it should be the case!